modules/updating-rhel-nodes-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="updating-rhel-nodes-restricted-network-cluster_{context}"]
+= Obtaining {product-title} packages
```

Comment:
Small thing, but can the anchor id for this heading be more similar to the heading/title wording? Just caught my attention.